### ISSUE TITLE
Error with Arrays.MultiLineArrayComma

### ISSUE THREAD

**Issue by yguedidi** [djoos/Symfony-coding-standard#45]
The following code will break, but it should be valid:

``` php
$foo = array('a' => true, 'b' => false, 'c' => array(
    'bar' => 1,
    'baz' => 2,
    'barbaz' => 3,
));
```

**Comment by lbayerl** [comment]
Unfortunately this is still the most annoying false-positive. Can we somehow support this or can you offer something to turn this rule off?

**Comment by wickedOne** [comment]
you can turn it off by adding the following to your ``ruleset.xml``
```xml
<rule ref="Symfony.Arrays.MultiLineArrayComma.Invalid">
    <severity>0</severity>
</rule>
```

**Comment by djoos** [comment]
Closing this one off; happy to discuss further if needed and take it from there!
